incubator/mariadb-galera/README.md

```diff
@@ -0,0 +1,115 @@
+# MariaDB Galera Cluster
```

Comment:
FYI @tongpu : I fixed the permission problem with this quite ugly hack:

https://github.com/extendedmind/charts/commit/f1455176470bd119d25ecb2cbc9ac4aa7a9ed566

This is related to:

https://github.com/kubernetes/kubernetes/issues/2630

where there is talk that fsGroup thing might also help, but this busybox-initContainer-chown-hack probably is more portable.